### DISCUSSION TITLE
Image display on Ender 3 v2 from ORCA

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#1208] [General]
Folks, been using this firmware for some time and love it.  

I am now trying ORCA slicer at the moment (from CURA) for no particular reason other than I can.  My issue is that I'm not sure how to add the pre-processing g-code (post-processing?) into ORCA to see the image displayed in the little Ender screen as I can see from having done this in CURA.   Not sure where to input the info in ORCA or is this not possible in ORCA?  

Many thanks, PC

**Comment by mriscoc** [comment]
Yes, it is possible, just check the guide: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-generate-a-gcode-preview

**Reply by Pridanc** [reply]
OK Miguel, thanks. I'll take a look.  Stay well!

**Comment by Pridanc** [comment]
Miguel,
I've tried hard to get ORCA to acknowledge the executable file provided by Giadej but anytime I try to then slice something it said "file doesn't exist" when I could see it.

So with my old brain I thought that I should put .exe on the end of the file name and YES it now works.

I amaze myself sometimes.
TY and Giadej very much.

​PC

Confidentiality
Statement & Notice: This email is covered by the Electronic Communications
Privacy Act, 18 U.S.C. 2510-2521 and intended only for the use of the
individual or entity to whom it is addressed. Any review, retransmission,
dissemination to unauthorized persons or other use of the original message and
any attachments is strictly prohibited. If you received this electronic
transmission in error, please reply to the above-referenced sender about the
error and permanently delete this message. Thank you for your cooperation.

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by Pridanc** [reply]
OK, never mind. Since ORCA would suddenly slice it, I thought I had made it work but no.  I can go back to CURA if I need pictures!
I've tried many options but none are working.
I have the most recent ORCA 1.9.0,  and latest Python 3.12.1
I'm going to stop trying now since computers are not really my thing. 
TY for trying to help. All the best, PC

**Reply by mriscoc** [reply]
If you use the exe provided by Giadej, the Python is irrelevant, so only put the path to that executable in Orca.
